### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by harrislapiroff**
Addresses (but does not completely resolve) #8

### CODE REVIEW

**Comment by marteki** [comment]
I reviewed this at Wagtail Sprint US 2018, and made some suggestions to clarify the install process. (Now added fixed in the previous commit!)

I also suggested adding images of the UI at a minimum to the docs (and GIFs to be better).

File: docs/basic_usage.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    the default value of ``True``) or multiple objects (with ``False``). The
+    default value is fine for drop-in
+    :class:`~wagtail:wagtail.wagtailadmin.edit_handlers.PageChooserPanel`
+    replacement.
+
+    .. note::
+        ``AutocompletePanel`` does not currently support receiving ``page_type``
+        as a list.
+        :class:`~wagtail:wagtail.wagtailadmin.edit_handlers.PageChooserPanel`
+        does take multiple page types like this.
```
Comment (tomdyson) [inline docs/basic_usage.rst]: Perhaps easier to read as

Unlike `wagtail.wagtailadmin.edit_handlers.PageChooserPanel`, `AutocompletePanel` does not support receiving `page_type` as a list.

File: docs/basic_usage.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    .. note::
+        ``AutocompletePanel`` does not currently support receiving the
```
Comment (tomdyson) [inline docs/basic_usage.rst]: Do we need 'currently'?

File: docs/customization.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+=============
+
+Wagtail Autocomplete provides the ability to customize the behavior of ``AutocompletePanel``.
+
+"Create New" Behavior
+=====================
+
+Sometimes you want users to not only be able to select pages or objects, but create new ones on the fly without leaving the object that they're currently editing. This can be particularly useful for tag-like objects, where you want to be able to add a tag with a particular title, even if that tag doesn't already exist in the database.
+
+You can enable this type of behavior by defining an ``autocomplete_create`` class method on your model. This method should accept a string value and return a new saved model instance:
```
Comment (tomdyson) [inline docs/customization.rst]: Wow!

**Comment by harrislapiroff** [comment]
Reading through the docs now I think I actually do want to add an example of using for multiple select before merging this PR.

File: docs/customization.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    .. code-block:: python
+
+        MyModel.objects.filter(my_special_field__icontains='part')
+
+    Additionally, this means that ``autocomplete_search_field`` *must* be a model field and cannot be an arbitrary property or method.
+
+Custom Label Display
+====================
+
+By default, the autocomplete widget will display the ``title`` field from a model. You can change this display by defining an ``autocomplete_label`` method on your model:
```
Comment (tomdyson) [inline docs/customization.rst]: change this display -> change this behaviour?

File: docs/getting_started.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    We do not currently support Wagtail 2.x, but we're working on it!
+
+Installation
+============
+
+Install with ``pip``:
+
+.. code-block:: sh
+
+    pip install https://github.com/emilyhorsman/wagtail-autocomplete/tarball/master/
```
Comment (tomdyson) [inline docs/getting_started.rst]: Could you upload it to PyPI?

File: docs/getting_started.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Setup
+=====
+
+Add ``'wagtailautocomplete'`` to your project's ``INSTALLED_APPS``. This enables Wagtail to auto-discover the Wagtail Autocomplete's admin hooks.
+
+Add Wagtail Autocomplete's url patterns to your project's url config, usually in ``urls.py``. This should still come before your ``wagtail_urls``:
```
Comment (tomdyson) [inline docs/getting_started.rst]: the Wagtail Autocomplete's -> Wagtail Autocomplete's
url patterns -> URL patterns
url config -> URL config
should still come -> should come

File: docs/using_other_models.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+==================
+
+:class:`AutocompletePanel` works with models other than
+:class:`~wagtail:wagtail.wagtailcore.Page` and subclasses of it.
+
+
+Selecting Snippets
+==================
+
+We have a Django model ``Link`` that we have registered as a snippet.
```
Comment (tomdyson) [inline docs/using_other_models.rst]: We have a Django -> For example, we have a Django

File: docs/getting_started.rst (hunk 5 above)
Comment (harrislapiroff) [reply docs/getting_started.rst]: Yes! That's the plan, but it'll be on a future branch. Tracking in #18

**Pull request merged**